compiler-utils/src/main/java/com/squareup/anvil/compiler/internal/reference/ClassReference.kt

```diff
@@ -220,6 +226,14 @@ public sealed class ClassReference : Comparable<ClassReference>, AnnotatedRefere
 
     override fun enclosingClassesWithSelf(): List<Psi> = enclosingClassesWithSelf
 
+    override fun getDescriptor(): ClassDescriptor {
+      return module.findClassAcrossModuleDependencies(this.classId)
```

Comment:
There's only one place that I'm a bit worried about. I'm not sure if there will be performance impact.
Should I cache class descriptor instead of resolving it every time?

Reply:
Extracted into https://github.com/square/anvil/pull/833